site/components/Form/Checkbox/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { InputHTMLAttributes } from "react";
+import { cx } from "@emotion/css";
+
+import { Text } from "@klimadao/lib/components";
+
+import * as styles from "./styles";
+
+interface Props {
+  inputProps: InputHTMLAttributes<HTMLInputElement>;
+  label: string;
+  errorMessage?: string;
+}
+
+export const Checkbox = React.forwardRef<HTMLInputElement, Props>(
```

Comment:
remind me, why do we need forwardRef for these?

Reply:
Hm, I followed the implementation of the other Pledge Inputs.
I think, these are not needed at all indeed.
Another follow-up refactoring PR? I added this to the list above.

Reply:
https://react-hook-form.com/faqs#Howtosharerefusage

Reply:
Here's another example from their documentation for further context.
At its core, we need to pass a ref onto the input elements so the library can work its magic.
It's up to us to agree on the pattern on how to do that moving forward.
https://react-hook-form.com/get-started#Integratinganexistingform

I think something to keep in mind here is that we're creating reusable form input elements to be used across the app. So imo, the example in the link above doesn't feel like the most scalable solution, but also the example is an extension on refs if we need to use refs for other purposes (eg. scroll into view)